dissector/messages.md

```diff
@@ -354,6 +354,10 @@ Message have zero to two payloads.
     </PtzControl>
     </body>
     ```
+    
+    - **Notes** : The only known values for speed are `32` for any movement and `0` for stop. The known movement
```

Comment:
There is a boolean flag in the ability list stored in the apps sqlite database that says supportsPTZSpeed or something like that. All by cameras are `false` so likely depends on the camera.

Reply:
Ok, then I will remove the note. I only have an Reolink E1 to try around and there the App only sends a direction command always with speed of 32 and a stop command with speed 0 as soon as the motion should end. I assumed that this was the case for all cameras.